LibationWinForms/ProductsGrid.cs

```diff
@@ -182,15 +182,7 @@ private async void liberate_Click(object sender, DataGridViewCellEventArgs e)
                 return;
             }
 
-            // not liberated: liberate
```

Comment:
I concede that your solution is probably what we need here.

The original idea was: I assumed if people clicked the stop light, it was because it's a big obvious thing and they didn't know about liberating their entire library since that's in a less obvious menu. To me 'full library' seems less obvious and more preferable. Sounds like my intuition hasn't been in sync with others' experience though.